modules/pxp-module-puppet

```diff
@@ -70,7 +70,8 @@ end
 
 def force_unicode(s)
   # Puppet still returns UTF-8 when locale is C or POSIX, so force encoding to that.
-  if s.encoding == Encoding::ASCII_8BIT
+  if !Puppet.features.microsoft_windows? &&
+    (s.encoding == Encoding::US_ASCII || s.encoding == Encoding::ASCII_8BIT)
```

Comment:
`ASCII_8BIT` is the same thing as `BINARY` - you get the actual bytes, with no encoding at all.